doc/cascadia/SettingsSchema.md

```diff
@@ -13,6 +13,7 @@ Properties listed below affect the entire window, regardless of the profile sett
 | `requestedTheme` | _Required_ | String | `system` | Sets the theme of the application. Possible values: `"light"`, `"dark"`, `"system"` |
 | `showTerminalTitleInTitlebar` | _Required_ | Boolean | `true` | When set to `true`, titlebar displays the title of the selected tab. When set to `false`, titlebar displays "Windows Terminal". |
 | `showTabsInTitlebar` | Optional | Boolean | `true` | When set to `true`, the tabs are moved into the titlebar and the titlebar disappears. When set to `false`, the titlebar sits above the tabs. |
+| `tabWidth` | Optional | String | `equal` | Sets the width of the tabs. Possible values: `"equal"`, `"sizeToContent"` |
```

Comment:
This doc suggests `sizeToContent`, though the rest of the pr says `sizetocontent`. I'm pretty sure we should go with the camelCase one